### ISSUE TITLE
Rails/Activerecord 7 Support

### ISSUE THREAD

**Issue by KidA001** [camertron/arel-helpers#49]
A gem I'm using has `arel-helpers` as a dependency. I was on Rails `7.0.0.alpha2 ` and had no issues with your gem or dependencies. Now that I upgraded to Rails `7.0.0` I get the following error

```
arel-helpers was resolved to 1.1.0, which depends on
        activerecord (~> 3.0)

    rails (~> 7.0.0) was resolved to 7.0.0, which depends on
      activerecord (= 7.0.0)
```

It looks like your gem requires `activerecord (>= 3.1.0, < 7)`. I'm not sure why it worked with `7.0.0.alpha2` but now it's throwing any error. Any plan to support `activerecord 7`?

**Comment by camertron** [comment]
Hey @KidA001, thanks for the issue :) I just merged https://github.com/camertron/arel-helpers/pull/51 and released v2.14.0 of arel-helpers with Rails 7 support. Let me know if you run into further issues 😄

**Comment by KidA001** [comment]
@camertron thank you!!!
